extensions/flyway/runtime/src/main/java/io/quarkus/flyway/runtime/FlywayDataSourceRuntimeConfig.java

```diff
@@ -108,6 +108,15 @@ public static final FlywayDataSourceRuntimeConfig defaultConfig() {
     @ConfigItem
     public boolean outOfOrder;
 
+    /**
+     * Ignore missing migrations when reading the history table. When set to true any migration present in the history table but
+     * absent in the configured locations will be ignored (and logged as a warning), when false (the default) the validation
+     * step
+     * will fail.
```

Comment:
Very minor but could you put `step` and `will fail` on the same line?

Reply:
Ah, I think it must have been the formatter (during the build) which changed this. I need to check my IntelliJ IDE settings.